src/components/SnippetCard/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import AceEditor from 'react-ace';
+
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+import { Button, Card, message, Modal, Typography } from 'antd';
+
+import 'ace-builds/src-noconflict/mode-c_cpp';
+import 'ace-builds/src-noconflict/mode-csharp';
+import 'ace-builds/src-noconflict/mode-css';
+import 'ace-builds/src-noconflict/mode-dart';
+import 'ace-builds/src-noconflict/mode-html';
+import 'ace-builds/src-noconflict/mode-java';
+import 'ace-builds/src-noconflict/mode-javascript';
+import 'ace-builds/src-noconflict/mode-json';
+import 'ace-builds/src-noconflict/mode-php';
+import 'ace-builds/src-noconflict/mode-python';
+import 'ace-builds/src-noconflict/mode-typescript';
+import 'ace-builds/src-noconflict/theme-monokai';
+
+// import CardDropdown from 'components/CardDropdown';
+
+import './styles.scss';
+
+const { Paragraph, Text } = Typography;
+const tagNames = ['Frontend', 'Web'];
+
+const tags = tagNames.map((tagName) => (
```

Comment:
lets put this within SnippetCard fn

Reply:
As a best practice, when we loop over a list of items and display them as elements, we extract them to their own component. Take a look at the example shown here in the docs
[Lists and Keys](https://reactjs.org/docs/lists-and-keys.html)